src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -66,18 +65,18 @@ export class SkyTabsetComponent
   public active: number | string;
 
   @Input()
-  public set queryParam(value: string) {
+  public set permalinkId(value: string) {
```

Comment:
Why `permalinkId` instead of `permalinkKey`?

Reply:
The meaning of `permalinkId` can stand on its own, and doesn't need to the companion input of `permalinkValue` to make sense.

<img width="699" alt="Screen Shot 2019-08-20 at 9 18 42 AM" src="https://user-images.githubusercontent.com/12497062/63350496-8c844700-c32b-11e9-8100-004270c814fa.png">

Reply:
Who's that guy? ☝️